extensions/store/sql/identity-hub-store-sql/src/main/java/org/eclipse/edc/identityhub/store/sql/SqlIdentityHubStore.java

```diff
@@ -68,9 +68,10 @@ public void add(IdentityHubRecord record) {
         });
     }
 
-    private IdentityHubRecord parse(ResultSet resultSet) throws SQLException {
+    IdentityHubRecord parse(ResultSet resultSet) throws SQLException {
```

Comment:
this method is `private`. making methods `package-private` or `protected` simply to be able to test them is an anti-pattern.